cypress/integration/parallel-2/updateContributionAmount.spec.ts

```diff
@@ -21,12 +24,12 @@ describe('Update contribution amount', () => {
 
 		cy.intercept('GET', '/api/me/mma?productType=Contribution', {
 			statusCode: 200,
-			body: [contributionCard],
+			body: toMembersDataApiResponse(contributionCard),
```

Comment:
Would it be better to update the fixtures to include the `user` and `products` properties rather than having to pipe them through a function to transform them?

Reply:
Interesting idea! I'm not sure how we would implement that as in productDetails.ts the products are all of type ProductDetails, so a const is needed to pass in the product into the MdapiResponse. Could you show me an example of what you mean? If it's a larger bit of refactoring to change all of the products I'm reluctant to do it part of this change as this should cover only what the refactor impacts

Reply:
I think this would make sense, but agree maybe a separate PR

Reply:
I think it should be done as the format of the fixtures no longer match what the API returns so they always have to be piped through `toMembersDataApiResponse`, but yeah, it could be a separate PR. We were going to go through the fixtures to update things like `subscriberId` with clearly mocked data, so perhaps it would make sense to do this at the same time?

Reply:
As discussed offline line I'll raise a ticket to cover reworking fixtures